src/evtx_chunk.rs

```diff
@@ -296,7 +303,7 @@ impl<'a> Iterator for IterChunkRecords<'a> {
 }
 
 impl<'a> Debug for EvtxChunk<'a> {
-    fn fmt(&self, fmt: &mut Formatter) -> Result<(), ::std::fmt::Error> {
+    fn fmt(&self, fmt: &mut Formatter) -> std::result::Result<(), ::std::fmt::Error> {
```

Comment:
Do we really need the custom `Debug` impls? 🙄

Reply:
🙄 (probably not)